packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -149,6 +153,10 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
         inlineImageOptions={props.inlineImageOptions}
         /* @conditional-compile-remove(mention) */
         mentionDisplayOptions={props.mentionOptions?.displayOptions}
+        /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
```

Comment:
It looks like the CC flag and assignment here is not needed, and the assignment is a duplication of the spread operator at line 145.

Reply:
good catch, updated